students/seungri/user/views.py

```diff
@@ -36,3 +37,29 @@ def post(self, request):
         except IntegrityError:
             return JsonResponse({"message": "DUPLICATE_ENTRY_KEY_ERROR"}, status=400)
 
+class UserLogIn(View):
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+            email           = data['email']
+            input_password  = data['password'].encode('utf-8')
+
+            if email == "" or input_password == "":
+                return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+            login_user  = User.objects.get(email=email)
+            db_password = login_user.password
+            db_password = db_password.encode('utf-8')
+            
+            if not bcrypt.checkpw(input_password, db_password):
+                return JsonResponse({"message": "INVALID_USER"}, status=401)
+
+            user      = {"user_id" : login_user.id}
+            token     = jwt.encode(user, SECRET_KEY, ALGORITHM)
```

Comment:
이 부분도 굳이 따로 변수를 생성할 필요까지는 없어보입니다.
```suggestion
            token     = jwt.encode({"user_id" : login_user.id}, SECRET_KEY, ALGORITHM)
```

Reply:
수정했습니다!